helm-chart-sources/pulsar/values.yaml

```diff
@@ -1072,6 +1082,12 @@ brokerSts:
     usePolicy: true
     maxUnavailable: 1
 
+## Pulsar Transactions metadata setup.
+## This is the job responsible to prepare the cluster for enabling transactions.
+## Job will start as soon as one broker pod is available.
+brokerTransactionsMetadata:
```

Comment:
If we are going to make this job it's own `component`, do you think we can make it just one job (instead of having one for `broker` and one for `brokerSts`)? I'd like to move away from the current model where we have two types of broker components because it makes the chart more complicated than it needs to be.

Reply:
that was my initial idea but then I figured out that they have different configs for affinity, tolerations... so I think it would be better to keep this component tightly coupled with the broker. 

What do you think ?